website/integrations/services/chronograf/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Integrate with Chronograf
+sidebar_label: Chronograf
+---
+
+# Chronograf
+
+<span class="badge badge--secondary">Support level: Community</span>
+
+## What is Chronograf
+
+> Chronograf lets you quickly visualize the data stored in InfluxDB, enabling you to build robust queries and alerts. It is simple to use and comes with templates and libraries for rapidly creating dashboards with real-time data visualizations.
+>
+> -- https://www.influxdata.com/time-series-platform/chronograf/
+
+## Preparation
+
+The following placeholders will be used:
+
+- `chronograf.company` is the FQDN of your Chronograf install.
+- `authentik.company` is the FQDN of your authentik install.
+
+## authentik configuration
+
+1. From the **authentik admin interface**, navigate to **Applications** -> **Applications** on the left sidebar.
```

Comment:
```suggestion
1. From the authentik Admin interface navigate to **Applications** -> **Applications** on the left sidebar.
```

Reply:
Sorry just saw this again, Yeah, consistency is a real bear. OK, I took a look at regular docs (not integrations) and we do not bold Admin interface. Bold is used for labels, buttons, field names, sections of the UI, enttries in the nav pane of the UI... but not when referring to the Admin interface in general. (I guess there could theoretically be a sentence whhere we tell users to click on the button called Admin interface in the UI, but...).

So yeah, let's not use bold for that.

Commas. I love commas and overuse them like crazy. They are almost a personal choice, at this point. You can keep that comma, it is friendly and causes no harm. 

Also thank you very much for all of your contributions to our integration guides.